VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -664,153 +527,219 @@ class VungleAdapter : PartnerAdapter {
                 }
             }
 
-            Vungle.loadAd(
-                request.partnerPlacement,
-                adm,
-                adConfig,
-                object : LoadAdCallback {
-                    override fun onAdLoad(id: String) {
-                        PartnerLogController.log(LOAD_SUCCEEDED)
-                        resumeOnce(
-                            Result.success(createPartnerAd(null, request))
-                        )
-                    }
+            fun loadVungleFullScreenAd(fullscreenAd: BaseFullscreenAd) {
+                fullscreenAd.apply {
```

Comment:
I don't think you need the apply here either.
fewer lines to just do fullscreenAd.adListener = and then load()

Reply:
Can do. 👍🏼

Reply:
Done.